index.js

```diff
@@ -13,9 +13,9 @@ try {
  * Defer.
  */
 
-var defer = typeof process === 'undefined' || typeof process !== 'function'
-  ? function(fn){ setTimeout(fn); }
-  : process.nextTick;
+var defer = typeof process !== 'undefined' && typeof process.nextTick === 'function'
```

Comment:
because i think i'm missing something, what would you suggest doing here instead? i'm more than happy to update this pr.